string robotName

float64 x
float64 y
float64 yaw

geometry_msgs/Vector3 angular
geometry_msgs/Vector3 linear